modules/c++/sys/unittests/test_symlink.cpp

```diff
@@ -67,18 +69,18 @@ TEST_CASE(testCreateSymlink)
     catch (const std::exception& ex)
     {
         std::cerr << "Caught std::exception: " << ex.what() << std::endl;
-        TEST_ASSERT(false);

```

Comment:
If this change is just to get rid of the compiler warning, hoping we can take care of that in build.py